GVFS/GVFS.Service/GvfsService.cs

```diff
@@ -26,6 +26,7 @@ public class GVFSService : ServiceBase
         private string serviceDataLocation;
         private RepoRegistry repoRegistry;

```

Comment:
Is the `GVFSService` class Window specific?

Reply:
Yes. Renamed it to `GVFSService.Windows.cs`